inspirehep/config.py

```diff
@@ -135,6 +135,114 @@ def _(x):
 USERPROFILES_EXTEND_SECURITY_FORMS = False
 USERPROFILES_SETTINGS_TEMPLATE = 'inspirehep_theme/accounts/settings/profile.html'
 
+# Collections
+# ===========
+COLLECTIONS_DELETED_RECORDS = '{dbquery} AND NOT collections.primary:"DELETED"'
+"""Enhance collection query to exclude deleted records."""
+
+COLLECTIONS_QUERY_PARSER = 'inspirehep.modules.search.parser:Main'
+"""User search query lexical parser."""
+
+COLLECTIONS_QUERY_WALKERS = [
+    'inspirehep.modules.search.walkers.pypeg_to_ast:PypegConverter',
+]
+"""Modules to create the query AST."""
+COLLECTIONS_USE_PERCOLATOR = False
+"""Define which percolator you want to use.
+
+Default value is `False` to use the internal percolator.
+You can also set True to use elasticsearch to provide percolator resolver.
+NOTE that ES percolator uses high memory and there might be some problems
+when creating records.
+"""
+
+COLLECTIONS_DEFINITION = [
+    {
+        "dbquery": "collections.primary:HEP",
```

Comment:
Since this config is under our control, you could write here `query` instead of `dbquery`, so that its sexier.